src/GLX/libglxgl.h

```diff
@@ -8,6 +8,48 @@
+    GLXContext (* ptr_glXCreateContext) (Display *dpy, XVisualInfo *vis, GLXContext shareList, Bool direct);
+    GLXPixmap (* ptr_glXCreateGLXPixmap) (Display *dpy, XVisualInfo *visual, Pixmap pixmap);
+    GLXContext (* ptr_glXCreateNewContext) (Display *dpy, GLXFBConfig config, int render_type, GLXContext share_list, Bool direct);
+    GLXPbuffer (* ptr_glXCreatePbuffer) (Display *dpy, GLXFBConfig config, const int *attrib_list);
+    GLXPixmap (* ptr_glXCreatePixmap) (Display *dpy, GLXFBConfig config, Pixmap pixmap, const int *attrib_list);
+    GLXWindow (* ptr_glXCreateWindow) (Display *dpy, GLXFBConfig config, Window win, const int *attrib_list);
+    void (* ptr_glXDestroyContext) (Display *dpy, GLXContext ctx);
+    void (* ptr_glXDestroyGLXPixmap) (Display *dpy, GLXPixmap pixmap);
+    void (* ptr_glXDestroyPbuffer) (Display *dpy, GLXPbuffer pbuf);
+    void (* ptr_glXDestroyPixmap) (Display *dpy, GLXPixmap pixmap);
+    void (* ptr_glXDestroyWindow) (Display *dpy, GLXWindow win);
+    const char * (* ptr_glXGetClientString) (Display *dpy, int name);
+    int (* ptr_glXGetConfig) (Display *dpy, XVisualInfo *visual, int attrib, int *value);
+    GLXContext (* ptr_glXGetCurrentContext) (void);
+    GLXDrawable (* ptr_glXGetCurrentDrawable) (void);
+    GLXDrawable (* ptr_glXGetCurrentReadDrawable) (void);
+    int (* ptr_glXGetFBConfigAttrib) (Display *dpy, GLXFBConfig config, int attribute, int *value);
+    GLXFBConfig * (* ptr_glXGetFBConfigs) (Display *dpy, int screen, int *nelements);
+    __GLXextFuncPtr (* ptr_glXGetProcAddress) (const GLubyte *procName);
+    __GLXextFuncPtr (* ptr_glXGetProcAddressARB) (const GLubyte *procName);
+    void (* ptr_glXGetSelectedEvent) (Display *dpy, GLXDrawable draw, unsigned long *event_mask);
+    XVisualInfo * (* ptr_glXGetVisualFromFBConfig) (Display *dpy, GLXFBConfig config);
+    Bool (* ptr_glXIsDirect) (Display *dpy, GLXContext ctx);
+    Bool (* ptr_glXMakeContextCurrent) (Display *dpy, GLXDrawable draw, GLXDrawable read, GLXContext ctx);
+    Bool (* ptr_glXMakeCurrent) (Display *dpy, GLXDrawable drawable, GLXContext ctx);
+    int (* ptr_glXQueryContext) (Display *dpy, GLXContext ctx, int attribute, int *value);
+    void (* ptr_glXQueryDrawable) (Display *dpy, GLXDrawable draw, int attribute, unsigned int *value);
+    Bool (* ptr_glXQueryExtension) (Display *dpy, int *errorb, int *event);
+    const char * (* ptr_glXQueryExtensionsString) (Display *dpy, int screen);
+    const char * (* ptr_glXQueryServerString) (Display *dpy, int screen, int name);
+    Bool (* ptr_glXQueryVersion) (Display *dpy, int *maj, int *min);
+    void (* ptr_glXSelectEvent) (Display *dpy, GLXDrawable draw, unsigned long event_mask);
+    void (* ptr_glXSwapBuffers) (Display *dpy, GLXDrawable drawable);
+    void (* ptr_glXUseXFont) (Font font, int first, int count, int list);
+    void (* ptr_glXWaitGL) (void);
+    void (* ptr_glXWaitX) (void);
+} __glXGLCoreFunctions;
+
 /**
```

Comment:
Analogous to above - the generator can produce all of the above already.

Mildly related: we might want to add PFN typedefs into the official header and drop all the local ones throughout LIBGLVND.